nussl/core/effects.py

```diff
@@ -129,7 +129,13 @@ class SoXFilter(FilterFunction):
     def __init__(self, filter_, **filter_kwargs):
         super().__init__(filter_, **filter_kwargs)
         if filter_ == "time_stretch":
-            self.func = lambda tfm: tfm.tempo(**filter_kwargs)
+            factor = filter_kwargs['factor']
+            if abs(factor - 1) <=  .1:
```

Comment:
Why not just `if .9 <= factor <= 1.1`?

Also, how were these bounds selected: something from PySox?

Reply:
Yes, these are the bounds PySox use to determine the recommended algorithm.

Reply:
`.9 <= factor <= 1.1` works too.

Reply:
I see now that your code matches what is in PySox:

https://github.com/rabitt/pysox/blob/master/sox/transform.py#L3192.

Okay cool, no change required then.